### PR TITLE
docs: move example inline into godoc

### PR DESCRIPTION
**Pull request by codyoss**
This change is needed, at least temporarily, to facilitate the
splitting of modules. The parent module can't depend on a module
that is about to be split out as tidying will fail to resolve
import paths as the code no longer exists.

### CODE REVIEW

File: doc.go [hunk 1]
```diff
@@ -28,18 +28,96 @@ Authentication and Authorization
 
 All the clients in sub-packages support authentication via Google Application Default
 Credentials (see https://cloud.google.com/docs/authentication/production), or
-by providing a JSON key file for a Service Account. See the authentication examples
-in this package for details.
+by providing a JSON key file for a Service Account. See examples below.
+
+Google Application Default Credentials(ADC) is the recommended way to authorize
```
Comment (tbpg) [inline doc.go:33]:
```suggestion
Google Application Default Credentials (ADC) is the recommended way to authorize
```

**Review by tbpg — CHANGES_REQUESTED** [review]
We can't do a 2-step process to avoid this? Publish the new module then update the root module?

**Comment by codyoss** [comment]
@tbpg I don't think so. Root module needs to be tagged first so that the sub-module can depend on it. Once tagged it can no longer reference the new child mods code without a new dependency on the child mod. I think we should avoid having the root depend on any gapic packages if possible.

**Review by tbpg — APPROVED** [review]
Ah, got it. Thanks for the explanation.

**Review by noahdietz — APPROVED** [review]

**Pull request merged**